Week3/homework/step2-3.js

```diff
@@ -2,46 +2,47 @@
 
 // Use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // eslint-disable-next-line prefer-const
-  let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
-  return result;
+	// eslint-disable-next-line prefer-const
+	let result = " ";
+	for (let i = 0; num > i; i++) {
+		result += str;
+	}
+	return result;
 }
 
+
 console.log('for', repeatStringNumTimesWithFor('abc', 3));
 
 // Use a 'while' loop
 function repeatStringNumTimesWithWhile(str, num) {
-  // eslint-disable-next-line prefer-const
-  let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
-  return result;
+	// eslint-disable-next-line prefer-const
+	let result = '';
+	while (num > 0) {
+		result += str;
+		num--;
+	}
+	return result;
 }
 
 console.log('while', repeatStringNumTimesWithWhile('abc', 3));
 
 // Use a 'do...while' loop
 function repeatStringNumTimesWithDoWhile(str, num) {
-  // eslint-disable-next-line prefer-const
-  let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
-  return result;
+	// eslint-disable-next-line prefer-const
+	let result = '';
+	do {
+		result += str;
+		num--;
+	} while (num > 0);
+
+	return result;
```

Comment:
Spacing feels a bit off here

Reply:
![Capture](https://user-images.githubusercontent.com/49612309/62277631-d8d41980-b446-11e9-9d07-8e6e2870ba68.JPG)


This is how it looks on my computer

Reply:
Strange...